aidial_adapter_bedrock/bedrock.py

```diff
@@ -36,6 +36,26 @@ async def acreate(cls, aws_client_config: AWSClientConfig) -> "Bedrock":
         )
         return cls(client)
 
+    @classmethod
+    def create(cls, aws_client_config: AWSClientConfig) -> "Bedrock":
+        client_kwargs = aws_client_config.get_boto_client_kwargs()
+        client_kwargs["service_name"] = "bedrock-runtime"
+        client = boto3.Session().client(**client_kwargs)
+        return cls(client)
+
+    async def aconverse_non_streaming(self, model: str, **params):
```

Comment:
You could improve the type hints:

1. Turn `ConverseParams` into `TypedDict`
2. Use [Unpack](https://peps.python.org/pep-0692/):

    ```python
    async def aconverse_non_streaming(self, model: str, **kwargs: Unpack[ConverseParams]):
        ...
    ```